lib/flavors.dart

```diff
@@ -0,0 +1,44 @@
+enum Flavor {
```

Comment:
@luongvo by using `flutter_config` and setting up `flavor` manually from #2 , we can move these configurations into `.env` file. This `flavors.dart` is redundant now 😁

Reply:
@manh-t yeah, I'm looking a way to get flavor state in main to set corresponding credentials for the network components. Sounds nice to go, try try 🙆‍♂️